web/server/codechecker_server/session_manager.py

```diff
@@ -244,6 +244,16 @@ def __init__(self, configuration_file, root_sha, force_auth=False):
                                 "... Disabling PAM authentication.")
                     self.__auth_config['method_pam']['enabled'] = False
 
+            if 'method_oauth' in self.__auth_config and \
+                    self.__auth_config['method_oauth'].get('enabled'):
+                if 'oauth' not in UNSUPPORTED_METHODS:
```

Comment:
Where is the point when `oauth` is added to the UNSUPPORTED_METHODS list?
If I am not mistaken this will always evaluate to true...

Reply:
fixed